### DISCUSSION TITLE
Plot Taxes

### DISCUSSION THREAD

**Discussion by rishireddy1599** [TownyAdvanced/Towny#6285] [Q&A] [answered]
Is it possible to set certain taxes on plots.

EX: plot 1 has a tax of 100 and plot 2 has a tax of 250

**Comment by LlmDl** [comment, marked as answer]
Normal plots, shop plots and embassy plots can have a tax set per-town.

**Reply by rishireddy1599** [reply]
can you change it to per plot basis?

**Reply by LlmDl** [reply]
You cannot.
